lib/app_home.dart

```diff
@@ -40,6 +43,51 @@ class AppHome extends StatefulWidget {
 }
 
 class _AppHomeState extends State<AppHome> {
+  @override
+  void initState() {
+    super.initState();
+    SplashWindowCloseNotifier.setWindowCloseHandler(
```

Comment:
maybe the showDialog functions could be extracted as showExitDialog() and showCustomExitDialog() or something like that to get them out of initState()?

Reply:
Done.